src/infrastructure/db/repositories/account.repository.ts

```diff
@@ -2,11 +2,7 @@ import { HttpException, Injectable } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
 import { PinoLogger } from 'nestjs-pino';
 import { AccountRegisterDto } from '../..//http/dtos';
-import {
-  DataBaseInternalError,
-  DataBaseUnathorizedError,
-  DataBaseUnprocesableContentError,
-} from '../errors';
+import { DataBaseInternalError, DataBaseUnathorizedError } from '../errors';
```

Comment:
```suggestion
import { DataBaseInternalError, DataBaseUnauthorizedError } from '../errors';
```
typo DataBaseUnathorizedError -> DataBaseUnauthorizedError (missing U in ..unaUth..)